test/integration/project-rows.test.js

```diff
@@ -0,0 +1,59 @@
+const SeleniumHelper = require('./selenium-helpers.js');
+
+const {
+    clickXpath,
+    findByXpath,
+    buildDriver
+} = new SeleniumHelper();
+
+let remote = process.env.SMOKE_REMOTE || false;
+let rootUrl = process.env.ROOT_URL || 'https://scratch.ly';
+
+if (remote) {
+    jest.setTimeout(60000);
+} else {
+    jest.setTimeout(10000);
+}
+
+let driver;
+
+describe('www-integration project rows', () => {
+    beforeAll(async () => {
+        driver = await buildDriver('www-integration project rows');
+        // driver.get(rootUrl);
+    });
+
+    beforeEach(async () => {
+        await driver.get(rootUrl);
+    });
+
+    afterAll(async () => await driver.quit());
+
+    test('Featured Projects row title', async () => {
+        let projects = await findByXpath('//div[@class="box"]/div[@class="box-header"]/h4');
+        let projectsText = await projects.getText();
+        await expect(projectsText).toEqual('Featured Projects');
+    });
+
+    test('Featured Project link', async () => {
+        await clickXpath('//div[@class="box"][descendant::text()="Featured Projects"]' +
+            '//div[contains(@class, "thumbnail")][1]/a[@class="thumbnail-image"]');
+        let gfOverlay = await findByXpath('//div[@class="stage-wrapper_stage-wrapper_2bejr box_box_2jjDp"]');
```

Comment:
When we talked in person I believe we decided to detect via the GUI class name instead of the calculated hash class name, which could change.

Reply:
I addressed this by searching for the div with id=guiPlayer instead of the green flag overlay.  I also changed the name of the file to make it slightly more accurate.

I updated the version of chromedriver and increased the timeouts for some tests which made them pass locally.